bootstrap/YakSetup.psm1

```
@@ -679,4 +679,37 @@ function Send-MagicPacket
   $UdpClient.Connect(([System.Net.IPAddress]::Broadcast), 9) # MagicPacket port
   $UdpClient.Send($MagicPacket, $MagicPacket.Length) | Out-Null
   $UdpClient.Close()
+}
+
+function Update-PSWTP_
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Update-PSWTP_' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)